qemu/tests/cfg/virtio_mem_numa_basic.cfg

```diff
@@ -0,0 +1,41 @@
+- virtio_mem_numa_basic:
+    only Linux
+    only Host_RHEL.m9
+    only RHEL.9
+    no s390x
+    type = virtio_mem_numa_basic
+    virt_test_type = qemu
+    login_timeout = 240
+    threshold = 0.025
+    smp = 8
+    aarch64:
+        vcpu_maxcpus = ${smp}
+    vcpu_maxcpus = 16
```

Comment:
If the case does not include vcpu hotplug, I would recommend `smp = vcpu_maxcpus`

Reply:
Updated, thanks!